packages/langium/src/lsp/node-kind-provider.ts

```diff
@@ -13,19 +13,35 @@ import { CompletionItemKind, SymbolKind } from 'vscode-languageserver';
 export interface NodeKindProvider {
     /**
      * Returns a `SymbolKind` as used by `WorkspaceSymbolProvider` or `DocumentSymbolProvider`.
+     * @param node AST node or node description.
+     * @returns {SymbolKind} - The corresponding symbol kind.
      */
     getSymbolKind(node: AstNode | AstNodeDescription): SymbolKind;
     /**
      * Returns a `CompletionItemKind` as used by the `CompletionProvider`.
+     * @param node AST node or node description.
+     * @returns {CompletionItemKind} - The corresponding completion item kind.
      */
     getCompletionItemKind(node: AstNode | AstNodeDescription): CompletionItemKind;
 }
 
+/**
+ * Default implementation of the `NodeKindProvider` interface.
+ * @remarks This implementation returns `SymbolKind.Field` for all nodes and `CompletionItemKind.Reference` for all nodes. Extend this class to customize symbol and completion types your langauge.
+ */
 export class DefaultNodeKindProvider implements NodeKindProvider {
-    getSymbolKind(): SymbolKind {
+    /**
+     * @inheritdoc
```

Comment:
Detail: `@inheritdoc` is not necessary in VS Code – it automatically copies the interface documentation if you don't override it.

Reply:
done - https://github.com/eclipse-langium/langium/pull/1580/commits/7e180e6028b78fbe041a1e42f522fff3b0b2bbe0